odl/operator/operator.py

```diff
@@ -707,6 +707,54 @@ def __call__(self, x, out=None, **kwargs):
                     raise_from(new_exc, err)
         return out
 
+    def norm(self, estimate=False, **kwargs):
```

Comment:
I think it would be good if it was possible to set the operator norm manually. The current framework would not allow for that.